lib/Mojolicious/Static.pm

```diff
@@ -34,12 +42,19 @@ sub dispatch {
   my $canon_path = join '/', @parts;
   return undef if $canon_path =~ /^\.\.\/|\\/ || !$self->serve($c, $canon_path);
   $stash->{'mojo.static'} = 1;
+
+  # Development assets will be rebuilt a lot, do not let browsers cache them
+  $c->res->headers->cache_control('no-cache')
+    if $c->app->mode eq 'development' && index($canon_path, $self->asset_dir) == 0;
+
   return !!$c->rendered;
 }
 
 sub file {
   my ($self, $rel) = @_;
 
+  $self->warmup unless $self->{index};
```

Comment:
It would be nice if there was an environment variable or something that allows you to reload the assets while developing, in case there's a new file added by the bundler while the server is running. Example: A background image in a CSS-file that gets processed by the bundler.

Reply:
I welcome improvements in this area.

Reply:
A very easy way to do this would be to turn running `warmup` more than once into a feature.

Reply:
As it happens, this already works for both, `$app->static->warmup` and `await app.static.warmup()`. There's just no tests yet. 😉